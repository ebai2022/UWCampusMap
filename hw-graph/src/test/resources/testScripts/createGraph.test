# Test graph creation

# Single graph creation
CreateGraph perkins

# Double graph creation
CreateGraph is
CreateGraph pretty
CreateGraph funny