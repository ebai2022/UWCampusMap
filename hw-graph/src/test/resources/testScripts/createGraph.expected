# Test graph creation

# Single graph creation
created graph perkins

# Double graph creation
created graph is
created graph pretty
created graph funny